### PR TITLE
Add visuals to 2021-09-30-data-maturity-open-source.md

### PR DESCRIPTION
**Pull request by hanisahkz**
This PR is for content, "[Leveraging Open Source Software to Build a Data Mature Ecosystem in the Social Sector: An Introduction](https://merlcenter.org/guides/data-maturity-open-source/)" and covers the following:

* **Main:** Add visuals to certain section
  * Spreadsheet for tracking can be found [here](https://docs.google.com/spreadsheets/d/15baCjqrkHPMRjrvJ599_7ToDnnQb7EElMxVRu1u6rX4/edit?usp=sharing).  
* **Others:**
  * Added anchor to `Table 1` to improve reading experience
  * Added improvement to use proper title case capitalization (based on APA [style guide](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case))
* See comparison table below for reference on how it gets rendered (the impacted area is highlighted in 🟪 :

| Current        |       PR       |
|----------------|:--------------:|
| Screenshot 2022-10-11 at 08-16-53 Leveraging Open Source Software to Build a Data Mature Ecosystem in the Social Sector An Introduction | Screenshot 2022-10-11 at 08-51-22 Leveraging Open Source Software to Build a Data Mature Ecosystem in the Social Sector An Introduction |

### CODE REVIEW

File: _guides/2021-09-30-data-maturity-open-source.md [hunk 1]
```diff
@@ -18,33 +18,36 @@ outgoing: false
...
 | ----------------------- | ------- |
-| [Data Orchard/DataKind UK’s Data Maturity Framework](https://drive.google.com/file/d/1wA_QKt49xzdxBguqQiFMu6VBgzEnCyZP/view) | Data Orchard’s and DataKind UK’s framework examines Data Maturity through seven lenses: leadership, skills, culture, data, tools, uses, and analysis. |
-| [CARE’s Responsible Data Maturity Model](https://www.ictworks.org/wp-content/uploads/2019/11/Responsible_Data_Maturity_Model_10-16-19.pdf) | CARE’s model provides a planning tool for organizations to evaluate their current data practices to identify resources and support for future efforts. In addition to examining elements, such as data and technology readiness, CARE emphasizes the need for “responsible” action to respond to the ethical, legal, social, and privacy-related challenges that come from using data. |
+| <img src="/assets/img/posts/v1-data-orchard-data-kind-mf.png" alt="Data Kind-Data Orchard maturity framework" width="250"> | [Data Orchard’s and DataKind UK’s framework](https://drive.google.com/file/d/1wA_QKt49xzdxBguqQiFMu6VBgzEnCyZP/view) examines data maturity through seven lenses: leadership, skills, culture, data, tools, uses, and analysis. |
```
Comment (hanisahkz) [inline _guides/2021-09-30-data-maturity-open-source.md:34]: I'm using HTML here as it provides better flexibility when customizing the visuals (size, position, etc). I noticed that some other content can also use this approach. Example: https://merlcenter.org/guides/open-source-merl-workflows/

**Review by ronna — COMMENTED** [review]
Looks good to me.

**Comment by scoker-me** [comment]
thanks. @hanisahkz  I think those changes look good in general but think we should change or move the text overlapping the infinity symbol so it's easier to read. was this made in Google Data Studio?

**Comment by hanisahkz** [comment]
> thanks. @hanisahkz I think those changes look good in general but think we should change or move the text overlapping the infinity symbol so it's easier to read. 

Agreed. I have updated the image used. In the blog, it will look like this:
 
image
 

> was this made in Google Data Studio?

Nope. For this one, I'm using Affinity Designer to re-create the loop and customize the design.

**Review by bonniewolfe — CHANGES_REQUESTED** [review]
Can we replace the MERL Center logo with the Care logo?
https://www.care.org/wp-content/uploads/2021/01/primary_logo_horizontal_min.svg

**Comment by hanisahkz** [comment]
> Can we replace the MERL Center logo with the Care logo? https://www.care.org/wp-content/uploads/2021/01/primary_logo_horizontal_min.svg

Sure! I've amended as requested. 

Just in case this wasn't considered - Initially, I wanted to add CARE's logo. However, I was unsure due to their T&C: [https://www.care.org/terms-of-use/](https://www.google.com/url?q=https://www.care.org/terms-of-use/&sa=D&source=editors&ust=1666802628665480&usg=AOvVaw2QAYw-yPmWksE4pXpdtDki). We can can email them about using their logo.

**Review by scoker-me — APPROVED** [review]

**Review by scoker-me — APPROVED** [review]

**Comment by scoker-me** [comment]
> > Can we replace the MERL Center logo with the Care logo? https://www.care.org/wp-content/uploads/2021/01/primary_logo_horizontal_min.svg
> 
> Sure! I've amended as requested.
> 
> Just in case this wasn't considered - Initially, I wanted to add CARE's logo. However, I was unsure due to their T&C: [https://www.care.org/terms-of-use/](https://www.google.com/url?q=https://www.care.org/terms-of-use/&sa=D&source=editors&ust=1666802628665480&usg=AOvVaw2QAYw-yPmWksE4pXpdtDki). We can can email them about using their logo.

Since our content is only for educational purposes, I think we can go ahead and include.

**Pull request merged**